Cleaning up

psomers3 opened the pull request:
I think this is a great thing to have done, so thanks a ton. I don't mind taking over the repository if you don't plan on keeping this up to date. I've changed the 'fetchscandata' to return python lists of the data instead of a string. And added a working live plotting script.

Commit:
inital commit

Commit:
cleaned up files

Commit:
removed build

Commit:
return data in python list format

Commit:
added plotting script and get data in xy

Commit:
moved examples up a level and deleted non-pertinent examples

thehapyone commented:
It's nice you have resurrected this project. Thanks for the cleanup, there were a lot of messy source codes. I realize you have dropped support for the myRplidar interface, is there a chance you could bring it back? 
The idea is to allow easy compatibility with existing codes using the SkoltechRobotics RPLidar Library, or what do you think?

psomers3 commented:
oh yea, sure. I didn't realize I deleted it. For compatibility, wouldn't it be cleaner to just call the Fastestrplidar class "RPlidar" and match the function names (and add more, such as what I added)?

thehapyone commented:
Yeah, that seems to make more sense. I just like the name "Fastestrplidar" though.
